content/blog/2025/09/kubewarden-1.29-release.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "Kubewarden 1.29 Release"
+authors:
+  - Víctor Cuadrado Juan
+date: 2025-09-30
+---
+
+Straight from the kitchen, Kubewarden 1.29 is served! This release is a medley
+bowl of healthy stack features, crisp policy improvements, and some fresh
+fixes, all seasoned with the wholesome flavour of paid-off tech debt.
```

Comment:
Well humored first paragraphs can be a new standard to our release blog post! :) 

I'm just afraid that I'll not be able to write them well as you do!